Smarty/templates/Header.tpl

```diff
@@ -212,25 +212,6 @@
 									</svg>
 									<span class="slds-assistive-text">{$APP.LNK_HELP}</span>
 							</button>
-							{if $SHOWQUICKCREATE}
```

Comment:
are you sure you have to delete this block?  I think this is the upper right corner quick create button and it is needed, or am I wrong?